terraform/modules/env-dashboards/dss-dashboard.tf

```diff
@@ -1,26 +1,133 @@
 locals {
   dss_dashboard = <<EOF
-{
-    "annotations": {
-        "list": [
-            {
-                "$hashKey": "object:267",
-                "builtIn": 1,
-                "datasource": "-- Grafana --",
-                "enable": true,
-                "hide": true,
-                "iconColor": "rgba(0, 211, 255, 1)",
-                "name": "Annotations & Alerts",
-                "type": "dashboard"
+        {
+            "annotations": {
+                "list": [
+                    {
+                        "$hashKey": "object:267",
+                        "builtIn": 1,
+                        "datasource": "-- Grafana --",
+                        "enable": true,
+                        "hide": true,
+                        "iconColor": "rgba(0, 211, 255, 1)",
+                        "name": "Annotations & Alerts",
+                        "type": "dashboard"
+                    }
+                ]
+            },
+            "editable": true,
+            "gnetId": null,
+            "graphTooltip": 0,
+            "iteration": 1543973320001,
+            "links": [],
+            "panels": [{
+            "aliasColors": {},
+            "bars": true,
+            "dashLength": 10,
+            "dashes": false,
+            "datasource": "${local.hca_logs_ttfb_datasource_name}",
+            "fill": 1,
+            "fillGradient": 0,
+            "gridPos": {
```

Comment:
you will need to ensure that this gridPos does not collide with others in the file, if you have inserted this into the file i believe you have to modify all the other grid pos for the other panels due to them shifting down. an alternative is to place the new panel at the bottom so you dont have to modify all the other gridpos.

Reply:
Yep.